src/net/bpfjit.c

```diff
@@ -604,9 +604,8 @@ emit_xcall(struct sljit_compiler *compiler, bpfjit_hint_t hints,
 		return status;
 
 	/* fn(buf, k, &err); */
-	/* Updated for latest version of sljit */
```

Comment:
This change is reverted to the original netbsd version and removed from our patch.
This part of the code is only built when used by the netbsd kernel, i.e there is a `#ifdef _KERNEL` a bit up,
so there is no need to patch the code here.